### ISSUE TITLE
(Automatically) (re)load external changes to SQLite database

### ISSUE THREAD

**Issue by mpsijm** [gristlabs/grist-core#836]
I was experimenting a bit with accessing a `.grist` SQLite database via the [`sqlite3`](https://www.sqlite.org/cli.html) program. Super awesome that it's possible to change data externally like this! However, I needed to restart Grist in order for the changes to become visible.

My setup:
- Arch Linux
- [grist-electron](https://github.com/gristlabs/grist-electron) v0.2.8, installed from [AUR](https://aur.archlinux.org/packages/grist-electron-bin)
- One table with some data columns and some formula columns

To reproduce:
- `sqlite3 *.grist`
- `INSERT INTO <table> (<data columns...>) VALUES (<values...>);`

Initially, nothing happens in Grist. However, after restarting Grist, the new row is shown. For this new row, the formula columns initially remain empty, but are updated within a few seconds, so that's really nice! However, it would be cool if Grist could reload external changes without a restart, either automatically or by adding a "reload database" button somewhere. Moving back to home and reopening the file didn't do the trick.

**Comment by wunter8** [comment]
There is an API endpoint to force the document to reload: `/api/docs/:docId/force-reload`

I haven't tested it myself, but I think if you edit the file with `sqlite3` and hit that endpoint (e.g., with `curl`), it will reload the document in the browser for any users currently looking at it

**Comment by mpsijm** [comment]
Thanks for the pointer! I tried `POST`ing to that endpoint with an empty body, as suggested by @paulfitz [here](https://community.getgrist.com/t/synchonization-with-sqlite-cli/3909/2), but unfortunately, that did not refresh the page in my Electron app. The OP of that thread suggest that this does work with the "Self-Managed Core" setup, which I will test later, since that's how I'd intend to use this anyway :slightly_smiling_face: 

Armed with a new search query "Grist force-reload", I also found [this thread](https://community.getgrist.com/t/grist-accepting-data-very-slowly/3417/11), which suggests clicking a skull icon in the settings page of the document, but I don't see this setting (potentially because I'm running from Electron).

In short, my initial request is most likely answered for the "Self-Managed Core" setup, and the question of force-reloading a Grist document in Electron remains.
